.github/workflows/ci-required.yml

```diff
@@ -105,3 +105,15 @@ jobs:
       - run: python scripts/doc_checker.py modin/experimental/core/execution/native/implementations/hdk_on_native/interchange/dataframe_protocol
       - run: python scripts/doc_checker.py modin/experimental/batch/pipeline.py
       - run: python scripts/doc_checker.py modin/logging
+
+  lint-black-isort:
```

Comment:
previously, the `isort` check was only triggered if the [modin's code was modified](https://github.com/modin-project/modin/blob/aeb7d0a82dd4902ecdcdba5f9cd3cf9f31979c76/.github/workflows/ci.yml#L11), that's why #6872 was showing green CI in PR but started failing after the merge